packages/theming/atlas/scripts/build.js

```diff
@@ -2,14 +2,27 @@ const { rm, cd } = require("shelljs");
 const { join } = require("path");
 const concurrently = require("concurrently");
 
-const mode = process.argv[2] || "production";
+const mode = process.argv[2] || "build";
 const MX_PROJECT_PATH = process.env.ATLAS_MX_PROJECT_PATH; // should be an absolute path.
-const outputDir =
-    mode !== "production" && MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "theme") : join(__dirname, "../dist/theme");
-const projectDeployDir = mode !== "production" && MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "deployment/web") : null;
+let outputDir;
+let projectDeployDir;
+
+switch (mode) {
```

Comment:
I think we should keep two modes/builds: `development` and `production`. To enable watch functionality, the option `--watch` needs to be passed as an argument. This feels more aligned with popular CLI commands.

Reply:
I disagree, this will misalign with our current way of doing things

Reply:
I feel like i am missing something here. Because if i look at the package.json of the atlas, i already see "start" "build" and "release".

Reply:
That's correct. I mean the way you call your `scripts/build.js` script. So instead of `node ./scripts/build.js development` for the start script in the `package.json` do this: `node ./scripts/build.js start`. The same kind of approach we do for widgets as you can see in my previous comment.

Reply:
Done